apps/builder/app/builder/features/breakpoints/breakpoints.tsx

```diff
@@ -133,28 +133,35 @@ export const Breakpoints = () => {
           )}
           {view === "selector" && (
             <>
-              {utils.breakpoints.sort(breakpoints).map((breakpoint) => {
-                return (
-                  <DropdownMenuCheckboxItem
-                    checked={breakpoint === selectedBreakpoint}
-                    key={breakpoint.id}
-                    onMouseEnter={() => {
-                      setBreakpointPreview(breakpoint);
-                    }}
-                    onMouseLeave={() => {
-                      setBreakpointPreview(selectedBreakpoint);
-                    }}
-                    onSelect={() => {
-                      selectedBreakpointIdStore.set(breakpoint.id);
-                    }}
-                  >
-                    {breakpoint.label}
-                    <DropdownMenuItemRightSlot>
-                      {breakpoint.minWidth}
-                    </DropdownMenuItemRightSlot>
-                  </DropdownMenuCheckboxItem>
-                );
-              })}
+              {Array.from(breakpoints.values())
+                .sort(sortMedia)
+                .reverse()
```

Comment:
What' wrong with default order direction?

Reply:
its just the order that makes visually sense and is aligned with webflow, while the sortMedia function sorts so that these are rendered in a correct order in the cssom

Reply:
So like from big to small for mobile first approach?